plugin/evm/orderbook/matching_pipeline.go

```diff
@@ -324,7 +324,7 @@ func areMatchingOrders(longOrder, shortOrder Order, marginMap map[common.Address
 		return nil
 	}
 
-	shortMargin := big.NewInt(0)
+	var shortMargin *big.Int = big.NewInt(0)
```

Comment:
what does this do?

Reply:
i was geting a warning on vs code 

<img width="500" alt="Screenshot 2024-01-17 at 8 28 30 PM" src="https://github.com/hubble-exchange/hubblenet/assets/22051142/3ad1711d-fe0a-4184-9d51-3e308cf0a4c0">